src/test/java/net/minecraftforge/test/TestCapabilityMod.java

```diff
@@ -160,5 +202,10 @@ public void readNBT(Capability<IExampleCapability> capability, IExampleCapabilit
         public String getOwnerType(){
             return "Default Implementation!";
         }
+        public boolean getVal() {
```

Comment:
@liach

You should really not worry about style of existing files in `minecraftforge/test` so much, it is already messed up.

If you want, can you make a PR to fix their style and disable a bunch of them that don't need to be running all the time? Then we have a good starting point and you can care about style from new PRs.